src/rag/rag.py

```
@@ -0,0 +1,183 @@
+        for i in range(0, len(cks), batch_count):
+            time.sleep(1)
+            batch = cks[i:i + batch_count]
+            try:
+                descriptions = [chunk.data_description for chunk in batch]
+                now_str = datetime.now().strftime("%Y-%m-%d %H:%M")
+                self.logger.info(f"Embedding: {min(i + batch_count, len(cks))}/{len(cks)} [{now_str}]")
+                embeddings = self.embed(descriptions)
+                self.vector_store.add(batch, embeddings) # type: ignore
+                now_str = datetime.now().strftime("%Y-%m-%d %H:%M")
+                self.logger.info(f"Finish: {min(i + batch_count, len(cks))}/{len(cks)} [{now_str}]")
+            except Exception as e:
```

Ruff (0.15.2)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)